schedule/yast/btrfs/btrfs+warnings.yaml

```diff
@@ -13,6 +13,7 @@ description: >
 vars:
   FILESYSTEM: btrfs
   PARTITIONING_WARNINGS: 1
+  INSTALLONLY: '1'
```

Comment:
HDDSIZEGB=40 . i assume that there is a reason that we use x2 of the default value.